src/assets/stylesheets/index.scss

```diff
@@ -1188,6 +1188,12 @@ section:focus >,
   transition:
     border-color 0.15s ease-in-out,
     box-shadow 0.15s ease-in-out;
+  &.dropzone {
+    cursor: pointer;
+    &.drag-active {
+      border: 0.15rem dashed;
+    }
```

Comment:
* [ ] Instead of new custom styling we could reuse `.form-control:focus`. I.e.:

```css
.form-control.drag-active,
.form-control:focus {
  color: #1e2022;
  background-color: #fff;
  border-color: rgba(140, 152, 164, 0.25);
  outline: 0;
  box-shadow: 0 0 1rem 0 rgba(140, 152, 164, 0.25);
}
```